x-pack/test/spaces_api_integration/security_and_spaces/apis/create.ts

```diff
@@ -22,7 +23,7 @@ export default function createSpacesOnlySuite({ getService }: FtrProviderContext
     expectConflictResponse,
     expectRbacForbiddenResponse,
     expectSolutionSpecifiedResult,
-  } = createTestSuiteFactory(esArchiver, supertestWithoutAuth);
+  } = createTestSuiteFactory(esArchiver, supertestWithoutAuth as unknown as SuperTest<any>);
```

Comment:
I had to do it to avoid test re-write: `SupertestWithoutAuthProviderType` won't allow to call `.auth(string,string)` with undefined arguments. I tried to rewrite it, but 2 configs started to fail, so I decide against it.